src/app/profile/user/details/details.component.scss

```diff
@@ -153,6 +153,9 @@
       font-size: 1.8rem;
       margin-bottom: 1rem;
     }
+    &--body {
+      font-size: 13px;
```

Comment:
Why is this required?

Reply:
Added it so that the font-size of characters during input remains same (13px) after the user saves the changes.